### PR TITLE
CLB + YSNC: Faldorn, Dread Wolf Herald + 7 cards

### LINKED ISSUE #543: Ancient Silver Dragon
https://discord.com/channels/267367946135928833/402524622958100501/980164638341754880

### LINKED ISSUE #673: Faldorn, Dread Wolf Herald

### LINKED ISSUE #674: Miirym, Sentinel Wyrm

### LINKED ISSUE #675: Ancient Bronze Dragon

### LINKED ISSUE #676: Delayed Blast Fireball

### LINKED ISSUE #677: Durnan of the Yawning Portal

### LINKED ISSUE #678: Agent of Raffine

### LINKED ISSUE #679: Bind to Secrecy

### PR DESCRIPTION
**Pull request by Suthro**
CLB: Added scripts for 4 CLB cards, including two legends:

- Faldorn, Dread Wolf Herald (closes #673)
- Miirym, Sentinel Wyrm (closes #674)
- Ancient Bronze Dragon (closes #675)
- Ancient Silver Dragon (script contributed in Discord by squee1968) (closes #543)

*** Update 6/12/22 8:02 PM:

CLB: Added scripts for 2 CLB cards:

- Durnan of the Yawning Portal (closes #677)
- Delayed Blast Fireball (closes #676)

*** Update 6/12/22 8:23 PM:

YSNC: Added scripts for 2 SNC alchemy cards:

- Agent of Raffine (closes #678)
- Bind to Secrecy (closes #679)

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/ancient_bronze_dragon.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Name:Ancient Bronze Dragon
+ManaCost:5 G G
+Types:Creature Elder Dragon
+PT:7/7
+K:Flying
+T:Mode$ DamageDone | ValidSource$ Card.Self | ValidTarget$ Player | CombatDamage$ True | Execute$ TrigRoll | TriggerDescription$ Whenever CARDNAME deals combat damage to a player, roll a d20. When you do, put X +1/+1 counters on each of up to two target creatures, where X is the result.
+SVar:TrigRoll:DB$ RollDice | ResultSVar$ Result | Sides$ 20 | SubAbility$ DBPutCounter
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/ancient_bronze_dragon.txt:7]: This one needs a reflexive trigger

File: forge-gui/res/cardsfolder/upcoming/ancient_silver_dragon.txt [hunk 2]
```diff
@@ -0,0 +1,12 @@
+Name:Ancient Silver Dragon
+ManaCost:6 U U
+Types:Creature Elder Dragon
+PT:8/8
+K:Flying
+T:Mode$ DamageDone | ValidSource$ Card.Self | Execute$ TrigRoll | CombatDamage$ True | ValidTarget$ Player | TriggerZones$ Battlefield | TriggerDescription$ Whenever CARDNAME deals combat damage to a player, roll a d20. 
+SVar:TrigRoll:DB$ RollDice | ResultSVar$ Result | Sides$ 20 | SubAbility$ DBImmediateTrigDraw
+SVar:DBImmediateTrigDraw:DB$ ImmediateTrigger | Execute$ TrigDraw | RememberSVarAmount$ Result | TriggerDescription$ Draw cards equal to the result.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/ancient_silver_dragon.txt:8]: But this one shouldn't have

File: forge-gui/res/cardsfolder/upcoming/ancient_bronze_dragon.txt (hunk 1 above)
Comment (Suthro) [reply forge-gui/res/cardsfolder/upcoming/ancient_bronze_dragon.txt]: Thanks for the catch. Shifted reflexive trigger to the correct script

File: forge-gui/res/cardsfolder/upcoming/bind_to_secrecy.txt [hunk 3]
```diff
@@ -0,0 +1,18 @@
+Name:Bind to Secrecy
+ManaCost:U B
+Types:Instant
+A:SP$ Charm | Choices$ CounterNonCreature,DBConjure | CharmNum$ 1 | SpellDescription$ If there are five or more mana values among cards in your graveyard, draft a card from CARDNAME's spellbook.
+SVar:CounterNonCreature:DB$ Counter | TargetType$ Spell | TgtPrompt$ Select target noncreature spell | ValidTgts$ Card.nonCreature | SubAbility$ TrigDraft | SpellDescription$ Counter target noncreature spell.
+SVar:DBConjure:DB$ MakeCard | TgtPrompt$ Select target creature card in an opponent's graveyard | ValidTgts$ Creature.OppOwn | TgtZone$ Graveyard | DefinedName$ Targeted | Zone$ Hand | RememberMade$ True | SubAbility$ DBEffect | SubAbility$ TrigDraft | SpellDescription$ Conjure a duplicate of target creature card in an opponent's graveyard into your hand. It perpetually gains "You may spend mana as though it were mana of any color to cast this spell."
+SVar:DBEffect:DB$ Effect | RememberObjects$ Remembered | StaticAbilities$ PerpetualAbility | Duration$ Permanent | Name$ Bind to Secrecy's Perpetual Effect | Triggers$ Update | SubAbility$ DBCleanup
```
Comment (Suthro) [inline forge-gui/res/cardsfolder/upcoming/bind_to_secrecy.txt:7]: A bug here slipped past me in scripting. The `DBEffect` is not correctly applying the perpetual cast-with-any-color effect to the conjured card. Working on a fix now

Comment (Suthro) [reply forge-gui/res/cardsfolder/upcoming/bind_to_secrecy.txt]: Resolved

**Comment by paulsnoops** [comment]
@Suthro I converted all the scripts to issues so they close once this has been merged

**Review by tool4ever — APPROVED** [review]

**Pull request merged**